Pillar_II/esm/src/dummy_prune_analysis.py

```diff
@@ -7,37 +11,40 @@
 from pycompss.api.parameter import IN  # type: ignore
 from pycompss.api.task import task  # type: ignore
 from pycompss.runtime.management.classes import Future  # type: ignore
-from hecuba import StorageDict
-import time
-import random
-
-class MetaDictClass(StorageDict):
-   '''
-   @TypeSpec dict <<keyname0:str>,valuename0:str>
-   '''
-
-def simulation_started(expid):
-    state = MetaDictClass.get_by_alias(expid) 
-    if len([k for k in s.keys()]) > 1:
-        return True
-    else:
-        return False
```

Comment:
This is not necessary, :point_up: 

We are calling this function that calls `state = MetaDictClass.get_by_alias(expid)` until it returns something. Then, after that we call `state = MetaDictClass.get_by_alias(expid)` again. We can simplify that. See below.